docs/source/concepts/service.rst

```diff
@@ -367,30 +367,35 @@ logic:
 
 .. code-block:: python
 
-    import typing as t
+    from typing import Any
```

Comment:
This is just a semantic change. It shouldn't be here.

Reply:
i think we should use `from typing import Any` instead `import typing as t` for clean code. I don't understand what's `t` when read the code below. using `from typing import Any` making it clear and explicit where the Any type is coming from.

Reply:
I think we did some check a while ago (or someone did this I don't remember exactly), but explicit import from typing is slower than the alias import, because you actually don't eagerly access the module `typing`.